bentoml/deployment/operator.py

```diff
@@ -31,7 +32,12 @@ def get_deployment_operator(deployment_pb):
 
         return SageMakerDeploymentOperator()
     elif operator == DeploymentSpec.AWS_LAMBDA:
-        from bentoml.deployment.serverless.aws_lambda import AwsLambdaDeploymentOperator
+        if config().getboolean('experiment', 'aws_lambda_v2'):
```

Comment:
discussed offline, let's remove this flag and swich to the sam based implementation